yb-voyager/cmd/importData.go

```diff
@@ -52,6 +52,7 @@ var dataFileDescriptor *datafile.Descriptor
 var truncateSplits bool                            // to truncate *.D splits after import
 var TableToColumnNames = make(map[string][]string) // map of table name to columnNames
 var valueConverter dbzm.ValueConverter
+var maxSplitSizeBytes int64
```

Comment:
Nit: Maybe you don't need to create a variable for this value. You can just call `tdb.MaxBatchSizeInBytes()` whenever required.